components/movementsensor/mpu6050/mpu6050.go

```diff
@@ -198,9 +201,10 @@ func NewMpu6050(
 			select {
 			case <-timer.C:
 				rawData, err := sensor.readBlock(sensor.backgroundContext, 59, 14)
+				// Record `err` no matter what: even if it's nil, that's useful information.
+				sensor.err.Set(err)
 				if err != nil {
 					sensor.logger.Infof("error reading MPU6050 sensor: '%s'", err)
```

Comment:
Do you still want this as an `Infof` level call?

Reply:
(I don't recall why we were logging this at the info level at all, though. Maybe that's leftover from debugging something, and should be removed entirely?)

Reply:
It should either be an Error level ( defeats the purpose of this PR) or a Debug level to hide it. Info will spam users with this error and defeat the purpose of what you're doing, so keeping it the same isn't useful

Reply:
@alan please address this.

Reply:
hehe, that's not my username on github. Apologies to Alan for confusing him!